timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/datastores/schemastore/dto/Predicate.java

```diff
@@ -61,10 +61,6 @@ public void registerSubject(long occurrences) {
     this.subjectsWithThisPredicate += occurrences;
   }
 
-  public boolean isOptional() {
```

Comment:
Why is optional removed?

Reply:
Ask Jauco why we did not need isOptional anymore here.